templates/sections/section-person-body-two-column.html.twig

```diff
@@ -11,6 +11,9 @@
 
   {% if content.right|render %}
     <div {{ region_attributes.right.addClass('right-region', right_class) }}>
+      <div class="node-stanford-person-su-person-contact">
+        <h2 class="block__title">Contact</h2>
+      </div>
```

Comment:
Do we want to make this conditional on anything being in the `content.right` container or is always displaying ok?

Reply:
We do have the `{% if content.right|render %}` and if there is no other content, the header doesn't appear. If there is any content, it should appear. It worked ok when I tested it locally. Can you think of a reason/case where it wouldn't?

Reply:
Good enough for me.